.rubocop_todo.yml

```diff
@@ -283,111 +163,310 @@ Style/MultilineMethodCallIndentation:
+# SupportedStyles: space, no_space
+# SupportedStylesForEmptyBraces: space, no_space
+Layout/SpaceInsideBlockBraces:
   Exclude:
-    - 'spec/**/*'
-    - 'app/helpers/container_steps_helper.rb'
+    - 'test/functionals/image_search_controller_test.rb'
+
+# Offense count: 11
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, EnforcedStyleForEmptyBraces, SupportedStylesForEmptyBraces.
+# SupportedStyles: space, no_space, compact
+# SupportedStylesForEmptyBraces: space, no_space
+Layout/SpaceInsideHashLiteralBraces:
+  Exclude:
+    - 'app/models/service/registry_api.rb'
+    - 'test/functionals/api/v2/containers_controller_test.rb'
     - 'test/functionals/containers_steps_controller_test.rb'
+    - 'test/functionals/image_search_controller_test.rb'
+    - 'test/test_plugin_helper.rb'
+    - 'test/units/image_search_service_test.rb'
 
 # Offense count: 2
 # Cop supports --auto-correct.
-Style/RedundantParentheses:
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: final_newline, final_blank_line
+Layout/TrailingBlankLines:
   Exclude:
-    - 'test/units/docker_registry_test.rb'
+    - 'app/services/foreman_docker/image_search.rb'
+    - 'app/views/api/v2/registries/main.json.rabl'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Layout/TrailingWhitespace:
+  Exclude:
+    - 'app/controllers/containers/steps_controller.rb'
+
+# Offense count: 1
+Lint/AmbiguousRegexpLiteral:
+  Exclude:
+    - 'test/functionals/api/v2/containers_controller_test.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Lint/UnifiedInteger:
+  Exclude:
+    - 'app/controllers/api/v2/containers_controller.rb'
+
+# Offense count: 16
+Metrics/AbcSize:
+  Max: 30
+
+# Offense count: 14
+# Configuration parameters: CountComments, ExcludedMethods.
+Metrics/BlockLength:
+  Max: 130
 
 # Offense count: 3
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 172
+
+# Offense count: 198
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 181
+
+# Offense count: 21
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 23
+
+# Offense count: 7
 # Cop supports --auto-correct.
-Style/RedundantSelf:
+# Configuration parameters: EnforcedStyle, SupportedStyles, Include.
+# SupportedStyles: action, filter
+# Include: app/controllers/**/*.rb
+Rails/ActionFilter:
   Exclude:
-    - 'app/models/concerns/foreman_docker/parameter_validators.rb'
+    - 'app/controllers/api/v2/containers_controller.rb'
+    - 'app/controllers/api/v2/registries_controller.rb'
+    - 'app/controllers/containers/steps_controller.rb'
+    - 'app/controllers/containers_controller.rb'
+    - 'app/controllers/registries_controller.rb'
+
+# Offense count: 10
+# Cop supports --auto-correct.
+Rails/ApplicationRecord:
```

Comment:
Since https://github.com/theforeman/foreman/pull/3729 was merged in core, this should really be fixed for the next minor version